README.md

```diff
@@ -100,7 +100,7 @@ The complete list of transformations supported and their usage in ImageKit can b
 | format                        | f                       |
 | radius                        | r                       |
 | background                    | bg                      |
-| border                        | bo                      |
```

Comment:
Please update SDK initialization section. Now we just need `urlEndpoint` to initialize the SDK. Please refer to Vue.js SDK for reference.

Reply:
hi @imagekitio 
added changes , check and let me know if there are any others 
thank you